enterprise/web.jsf20/external/binaries-list

```diff
@@ -1 +1 @@
-99277566601C4D5C2598B0206B5DC071932F3641 jsf-2.2.zip
+94AE75EC83EF51C2F94C17D7C0F85E7C6518CC71 org.glassfish:javax.faces:2.2.7
```

Comment:
Missing the license header?

Reply:
@junichi11 , Thanks for pointing this out. It was an oversight on my part.